### PR TITLE
fix: GOLANGCI_LINT_VERSION

### PR DESCRIPTION
**Pull request by dcoppa**
Without this, I get an error while building the master branch of argo-cd:

go: downloading gopkg.in/ini.v1 v1.67.0
go: downloading github.com/magiconair/properties v1.8.7
go: downloading github.com/pelletier/go-toml/v2 v2.0.6
go: downloading golang.org/x/text v0.7.0
+ export DOWNLOADS=/tmp/dl
+ DOWNLOADS=/tmp/dl
+ export BIN=/usr/local/bin
+ BIN=/usr/local/bin
+ mkdir -p /tmp/dl
+ ARCHITECTURE=
+ case $(uname -m) in
++ uname -m
+ ARCHITECTURE=amd64
+ INSTALL_OS=
++ uname -s
+ unameOut=Linux
+ case "${unameOut}" in
+ INSTALL_OS=linux
+ for product in $*
++ dirname ./install.sh
+ ARCHITECTURE=amd64
+ INSTALL_OS=linux
+ ./installers/install-lint-tools.sh
+ GOLANGCI_LINT_VERSION=1.61.0
+ GO111MODULE=on
+ go install github.com/golangci/golangci-lint/cmd/golangci-lint@1.61.0
go: github.com/golangci/golangci-lint/cmd/golangci-lint@1.61.0: github.com/golangci/golangci-lint/cmd/golangci-lint@1.61.0: invalid version: unknown revision 1.61.0
building at STEP "RUN ./install.sh helm &&     ./install.sh kustomize &&     ./install.sh codegen-tools &&     ./install.sh codegen-go-tools &&     ./install.sh lint-tools &&     go install github.com/mattn/goreman@latest &&     go install github.com/kisielk/godepgraph@latest &&     go install github.com/jstemmer/go-junit-report@latest &&     rm -rf /tmp/dl &&     rm -rf /tmp/helm &&     rm -rf /tmp/ks_*": while running runtime: exit status 1

make: *** [Makefile:276: test-tools-image] Error 1

### CODE REVIEW

File: hack/installers/install-lint-tools.sh [hunk 1]
```diff
@@ -2,6 +2,6 @@
 set -eux -o pipefail
 
 # renovate: datasource=go packageName=github.com/golangci/golangci-lint
-GOLANGCI_LINT_VERSION=1.61.0
+GOLANGCI_LINT_VERSION=v1.61.0
 
 GO111MODULE=on go install "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}"
```
Comment (ggjulio) [inline hack/installers/install-lint-tools.sh:7]: It was my fault :
- https://github.com/argoproj/argo-cd/commit/d17aafd19a465fb9992d8189381adc3bdd488ead#diff-379bb3898b60010b7127603fcc9dcfb4d7f8e921997c1b43eaba78c5a99ce3c8L4  

The `v` should be outside the VM else `renovate` will either fail or won't stop removing it.
```diff
- GOLANGCI_LINT_VERSION=v1.61.0
+ GOLANGCI_LINT_VERSION=1.61.0

- GO111MODULE=on go install "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}"
+ GO111MODULE=on go install "github.com/golangci/golangci-lint/cmd/golangci-lint@v${GOLANGCI_LINT_VERSION}"

Comment (dcoppa) [reply hack/installers/install-lint-tools.sh:7]: You're right, "v" should be outside.
Thanks

Comment (ggjulio) [reply hack/installers/install-lint-tools.sh]: thank you for fixing it

**Review by ggjulio — APPROVED** [review]

**Review by crenshaw-dev — APPROVED** [review]
Thanks!

**Pull request merged**